src/topology/algebra/big_operators/ring.lean

```diff
@@ -0,0 +1,243 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+-/
+import topology.algebra.big_operators.basic
+import topology.algebra.ring
+
+/-!
+# Infinite sum in a ring
+
+This file provides lemmas about the interaction of infinite sums and (scalar) multiplication.
+-/
+
+open filter finset function
+open_locale big_operators classical
+
+variables {R α β γ : Type*}
+
+section const_smul
+variables [monoid R] [topological_space α] [add_comm_monoid α] [distrib_mul_action R α]
+  [has_continuous_const_smul R α] {f : β → α}
+
+lemma has_sum.const_smul {a : α} (r : R) (hf : has_sum f a) : has_sum (λ z, r • f z) (r • a) :=
+hf.map (distrib_mul_action.to_add_monoid_hom α r) (continuous_const_smul r)
+
+lemma summable.const_smul (r : R) (hf : summable f) : summable (λ z, r • f z) :=
+(hf.has_sum.const_smul _).summable
+
+lemma tsum_const_smul [t2_space α] (r : R) (hf : summable f) : ∑' z, r • f z = r • ∑' z, f z :=
+(hf.has_sum.const_smul _).tsum_eq
+
+end const_smul
```

Comment:
These ones possibly feel out of place, as they're not about rings at all. (They're also precisely the lemmas that _will_ multiplicativize). So I'd lean towards putting them back in `basic`.